tests/experimental/datasources/test_spark_datasource.py

```diff
@@ -62,6 +62,18 @@ def test_construct_spark_filesystem_datasource(
 @pytest.mark.unit
 def test_add_csv_asset_to_datasource(
     spark_filesystem_datasource: SparkFilesystemDatasource,
+):
```

Comment:
@NathanFarmer This module is being renamed in #7197 (by the way, the method `test_add_csv_asset_to_datasource()` seems to already exist; is it being simply moved?).